src/HM1.java

```diff
@@ -8,7 +8,8 @@ public static int MYMETHOD(int a, int b) {
         }
 
         public static void main(String[] args) {
-            MYMETHOD(4, 6);
+            int result = MYMETHOD(4, 6);
```

Comment:
method name should reflect the sense of the method
also, it should be named according to Java code convention - lower camel case
sumOfTwoNumbers - please rename the method

Reply:
Rename Year to year (we should name variables starting lower case)

Reply:
Class name should start from Upper case letter - instead of myClass please name as MyClass